### DISCUSSION TITLE
with-nativewind starter repo failing to deploy to Vercel

### DISCUSSION THREAD

**Discussion by adherb** [nandorojo/solito#122] [Q&A]
I am trying to deploy a new project to my Vercel account using the ```/with-tailwind``` example repo.

The issue is when I try and set ```ENABLE_ROOT_PATH_BUILD_CACHE``` to equal ```1```, under the 'Configure project' subtitle, I get an error and the project fails to deploy.

The error under the 'Deploy' subtitle reads 'An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.'

I have tried this several times, but I'm still unable to get it to work.

Does anybody have any ideas why this would be?

**Comment by nandorojo** [comment]
Could you comment on #97? This PR is still a work-in-progress, but it should get stable soon after we complete #120
